lib/topological_inventory/ansible_tower/operations/core/topology_api_client.rb

```diff
@@ -18,8 +18,8 @@ def update_task(task_id, state:, status:, context:)
           end
 
           def svc_instance_url(service_instance)
-            rest_api_path = '/service_instances/{id}'.sub('{' + 'id' + '}', service_instance&.id.to_s)
-            topology_api_client.api_client.build_request(:GET, rest_api_path).url
+            return if service_instance.nil?
+            topology_api_client.api_client.build_request(:GET, "/service_instances/#{service_instance.id}").url
```

Comment:
I need to check but I think this needs to be `.external_url` not `.url`

Reply:
We don't even need this method, we're given a TopologicalInventoryApiClient::ServiceInstance object as an argument which already has `#external_url` as an attribute.